src/views/HousingLottery/index.jsx

```diff
@@ -1,7 +1,99 @@
+import React, { useState } from 'react';
+import {
+  Input,
+  Button,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+  Card,
+  CardHeader,
+  CardContent,
+  Grid,
+} from '@mui/material';
+import housingService from 'services/housing';
+import styles from './HousingLottery.module.css';
 import PreferredHall from './studentView/PreferredHall';
 
 const HousingLottery = () => {
-  return <PreferredHall />;
-};
```

Comment:
I guess you get these two lines from merging from senior-project. You shouldn't remove these lines. If you remove them, the preferred hall box will disappear. Instead, you should keep both these lines and your code for prefer box. It seems like you also already deleted the files related to PreferredHall component. I would recommend you merge from senior-project again to get those files. Does that make sense?